static/js/util/redux.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+import type {
+  ActionCreator,
+  AsyncActionHelper,
+  AsyncActionCreator
+} from '../flow/reduxTypes';
+import type { Dispatch } from 'redux';
+
+export function createActionHelper(dispatch: Dispatch, actionCreator: Function): (...args: any) => void {
+  return (...args) => dispatch(actionCreator(...args));
+}
+
+export type ActionHelpers = Array<{[k: string]: (...args: any) => void}>;
+export function createSimpleActionHelpers(dispatch: Dispatch, actionManifest: ActionCreator[]): ActionHelpers {
+  return actionManifest.map(actionCreator => (
+    { [actionCreator.name]: createActionHelper(dispatch, actionCreator) }
+  ));
+}
+
+export type AsyncActionHelpers = Array<{[k: string]: AsyncActionHelper}>;
+export function createAsyncActionHelpers(dispatch: Dispatch, actionManifest: AsyncActionCreator[]): AsyncActionHelpers {
+  return actionManifest.map(actionCreator => (
+    { [actionCreator.name]: createActionHelper(dispatch, actionCreator) }
+  ));
+}
```

Comment:
We could turn all the action creators into async ones, but we'd have to refactor all of the synchronous ones to make that work (which seems out of scope for this pr). I think we also rely on synchronicity for a lot of them, so I imagine that would be a really big refactor.

agree re: actionManifest, it's not really a manifest any more, haha.